CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
-cmake_minimum_required(VERSION 3.17)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
Please do not increase the required version unless this is required. With 3.17, I can compile chromo on our cluster, but not with 3.20. The idea here is to use the lowest version that actually works.

Reply:
I will try with 3.17. I changed it to 3.20 when I tried to fix the problem with search of the right python binary (find_package(Python...)).